chapters/ch13.md

```diff
@@ -1,40 +1,26 @@
 # 13. You want to build a new email anti-spam system. Your team has several ideas:
-->
-
+-> Bạn mong muốn xây dựng một hệ thống phòng chống email rác mới. Đội của bạn có một vài ý tưởng:
 
 * Collect a huge training set of spam email. For example, set up a “honeypot”: deliberately send fake email addresses to known spammers, so that you can automatically harvest the spam messages they send to those addresses.
-->
-
+-> Thu thập một tập huấn luyện lớn về email rác. Ví dụ như thiết lập một Honeypot: cố ý gửi các địa chỉ email giả đến những spammer đã biết, và bạn có thể thu thập các tin nhắn rác mà họ gửi đến địa chỉ đó một cách tự động.
```

Comment:
Mình thấy "honeypot" hoàn toàn có thể dịch là "hũ mật" :D

Reply:
Nghe có vẻ thú vị nhỉ :D từ này mình thấy nó mô tả một khái niệm nên mình lại không muốn dịch ra như thế :D

Reply:
Honeypot là một khái niệm ở trong security domain. Trong domain đó nhắc đến honeypot thì cũng như trong người làm về ML nhắc đến overfit. Theo mình không nên dịch.

Reply:
> Honeypot là một khái niệm ở trong security domain. Ở đó nó quen thuộc giống như từ overfit đối với những ai biết về ML. Theo mình không nên dịch.

Vậy thì phần này cần được giải thích thêm trong glossary. Em cũng không hiểu từ này nghĩa là gì, giống như người ngoài ngành nhìn "overfitting" cũng không hiểu.

Em sẽ tìm cách phù hợp để có thêm giải nghĩa của người dịch trong bài.

Reply:
"Honeypot" dịch là "hũ mật" thì sát nghĩa nhưng ko hiểu đc ngay. Dịch nôm na hơn chút thì có thể là "Mật ngọt nhử duồi" hay "Mồi nhử" để thu hút spam emails từ đó có training data.  Đúng là nên để nguyên "Honeypot" và có mở ngoặc giải thích thì ae trong ML mới hiểu nhanh đc. Chứ để nguyên không giải thích thì không phải ai cũng hiểu đc.

Reply:
theo mình từ gần nghĩa nhất với honeypot ở trong ngữ cảnh này là từ decoy, dịch ra là "bẫy". Mình gợi ý từ decoy để nếu muốn dịch ra tiếng việt thì có thêm option để dịch. Ý nghĩa của honeypot thì bạn sonvx đã giải thích rất rõ ở trên.

Reply:
Theo mình thì từ mồi nhử (số) của bạn sonvx đề xuất là rất gần với nghĩa gốc của từ "honeypot" rồi. Chỉ là trong glossary nên thêm context để mọi người hiểu thôi.